qiskit/transpiler/passes/optimization/echo_rzx_weyl_decomposition.py

```diff
@@ -38,7 +35,7 @@ class EchoRZXWeylDecomposition(TransformationPass):
     Each pair of RZXGates forms an echoed RZXGate.
     """
 
-    def __init__(self, backend: basebackend):
+    def __init__(self, backend: BaseBackend):
```

Comment:
This has the same bug as the other pass, I guess I'll bundle this as one more critical bugfix PR before the release....

Reply:
Resolved with #7331